### PR TITLE
Implement Stonecoil Serpent

### PR DESCRIPTION
**Pull request by tsirides**

### CODE REVIEW

File: Mage.Sets/src/mage/cards/b/BlacklanceParagon.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import mage.filter.common.FilterCreaturePermanent;
+import mage.target.common.TargetCreaturePermanent;
+
+/**
+ *
+ * @author LevelX2
+ */
+public final class BlacklanceParagon extends CardImpl {
+
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent(SubType.KNIGHT, "knight");
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/b/BlacklanceParagon.java]: Knight needs to be capitalized

File: Mage.Sets/src/mage/cards/b/BlacklanceParagon.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(1);
+
+        // Flash
+        this.addAbility(FlashAbility.getInstance());
+
+     
+        // When Blacklance Paragon enters the battlefield, target Knight gains deathtouch and lifelink until end of turn. 
+        Effect effect = new GainAbilityTargetEffect(DeathtouchAbility.getInstance(), Duration.EndOfTurn);
+        effect.setText("target knight gains deathtouch");
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/b/BlacklanceParagon.java]: also here

File: Mage.Sets/src/mage/cards/b/BlacklanceParagon.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.SubType;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.target.common.TargetCreaturePermanent;
+
+/**
+ *
+ * @author LevelX2
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/b/BlacklanceParagon.java]: Don't forget to give yourself credit!

Comment (tsirides) [reply Mage.Sets/src/mage/cards/b/BlacklanceParagon.java]: Yeah submitted new one ;) Thanks!!

**Pull request merged**